_config.yml

```diff
@@ -1,3 +1,3 @@
-baseurl: /honeybee-design-system
+baseurl: /it-design-system
```

Comment:
I *think* this would need to be done in coordination with also renaming the GitHub repo to `it-design-system`?

Reply:
Yes, but that can take place if this is merged. For anyone who'd cloned the repo locally they'd have to fix their git remote URL